Mathlib/Algebra/Algebra/Unitization.lean

```diff
@@ -749,6 +749,19 @@ theorem starLift_symm_apply_apply (φ : Unitization R A →⋆ₐ[R] C) (a : A)
     Unitization.starLift.symm φ a = φ a :=
   rfl
 
+/-- If `φ : A →⋆ₙₐ[R] C` is injective, the natural lift to `Unitization R A →⋆ₐ[R] Unitization R C`
+is also injective. -/
+lemma starLift_injective {R A C : Type*} [CommRing R] [StarRing R] [NonUnitalRing A]
+    [StarRing A] [Module R A] [SMulCommClass R A A] [IsScalarTower R A A] [NonUnitalRing C]
+    [Module R C] [IsScalarTower R C C] [SMulCommClass R C C] [StarRing C] [StarModule R C]
+    {φ : A →⋆ₙₐ[R] C} (hφ : Function.Injective φ) :
+    Function.Injective (starLift <| (inrNonUnitalStarAlgHom R C).comp φ) :=
```

Comment:
Should `starLift <| (inrNonUnitalStarAlgHom R C).comp φ` be called `starMap φ`?

Reply:
I've added `starMap` and some basic API for it.